Translate blog posts from Cantonese to English: scheme, table, and simplified (#94)
laubonghaudoi/jyutping.org: src/content/blog/en/simplified.mdx

---
title: Why are Simplified Characters unsuitable for writing Cantonese?
linkTitle: Why are Simplified Characters unsuitable for writing Cantonese?
url: /en/blog/simplified
weight: 4
description: |

>   The Simplified Chinese Character scheme was designed specifically for Mandarin and has many problems when used to write Cantonese.

Reviewer: ```suggestion
  The Simplified Chinese Character scheme was designed specifically for Mandarin and is problematic when used to write Cantonese.
```

---
title: Why are Simplified Characters unsuitable for writing Cantonese?
linkTitle: Why are Simplified Characters unsuitable for writing Cantonese?
url: /en/blog/simplified
weight: 4
description: |
  The Simplified Chinese Character scheme was designed specifically for Mandarin and is problematic when used to write Cantonese.
---

import Alert from "@/components/Alert.tsx";

> The [Chinese Character Simplification Scheme](https://en.wikipedia.org/wiki/Simplified_Chinese_characters) currently in use in mainland China was formulated by the Committee for Research on Chinese Language Reform in the 1950s. Its design inherited from the simplified character tables of the National Language Unification Preparatory Committee. Among these simplified characters, some were simplified based on the principle of [merging homophones](https://en.wikipedia.org/wiki/Merge_of_homophonous_characters_in_simplified_Chinese), serving as a transitional step toward the ultimate goal of [Romanization of Chinese](https://en.wikipedia.org/wiki/Romanization_of_Chinese).

Reviewer: Italicise the scheme name.
